components/block/element.js

```diff
@@ -84,21 +148,19 @@ export default class Block extends LitElement {
   }
 
   renderLinkedBlockMenu() {
-    return html`
-    <button class="refresh-btn" @click=${this.refresh}>
+    return html` <button class="refresh-btn" @click=${this.refresh}>
```

Comment:
Reformat it like this:
```js
return html`
  <button class="refresh-btn" @click=${this.refresh}>
    ${this.refreshIcon()}
  </button>
`;
```

Add new line after html` and before ` and prettier will reformat it like above

Reply:
Done https://github.com/netgen-layouts/layouts-ui-components/pull/2/commits/41b03365f396fa39af17bf90f4911f75540e1d01